CHANGES.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased #unstable
 
+ * opt-in for statistics collection (shared with TLC and TLA+ Toolbox), see #288
```

Comment:
Can we put this under a subsection (like we have for the Documentation? I think it was requested that we have a bit more structured release notes, to make it clear what kinds of fixes/changes. So maybe this would go under `### Features` or something?